packages/contracts-rfq/contracts/interfaces/IFastBridgeV2.sol

```diff
@@ -4,6 +4,21 @@ pragma solidity ^0.8.20;
 import {IFastBridge} from "./IFastBridge.sol";
 
 interface IFastBridgeV2 is IFastBridge {
+    enum BridgeStatus {
+        NULL, // doesn't exist yet
+        REQUESTED,
+        RELAYER_PROVED,
+        RELAYER_CLAIMED,
+        REFUNDED
+    }
+
+    struct BridgeTxDetails {
+        BridgeStatus status;
+        uint40 proofBlockTimestamp;
+        uint48 proofBlockNumber;
```

Comment:
This is purely hypothetical, but wouldn't using `uint48` for timestamps and `uint40` for block numbers make more sense here? The EVM chains have under 1 block/second, so `block.number < block.timestamp`.

Again, this is purely hypothetical, as overflowing these values is sooo far in the future.

Reply:
my thought was that seconds timestamp is the more behaviorally predictable of the two and uint40 covers it sufficiently - so whatever is remaining can just go to Block #. AFAIK an L2 can technically decide to do whatever they want with their block #s, so overflow seemed more possible there (although in either case, the possibility is so remote to be basically nothing)

Reply:
That's a very good point